Integrations/HelloWorld/HelloWorld.yml

```diff
@@ -0,0 +1,55 @@
+category: Authentication
```

Comment:
perhaps different category? utils ?

Reply:
it doesn't matter. Because it is HelloWorld